packages/vue/__tests__/vfor.spec.ts

```diff
@@ -0,0 +1,45 @@
+import * as Vue from '../src'
+import { createApp, reactive, onMounted } from '../src'
+import { nextTick } from '../../runtime-test/src'
+;(window as any).Vue = Vue
```

Comment:
I am not sure if this is a correct approach for testing here. Let's wait for Evan's inputs.

Reply:
Thanks for the feedback, I improved a little bit the test. 

But let's wait for him since I have my doubts about if the test is placed correctly in the project structure. because it is a combined behavior with `v-for`, `@click` and `runtime-dom`.